examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -268,7 +277,7 @@ class CSS2DRenderer {
 
 			scene.traverseVisible( function ( object ) {
 
-				if ( object.isCSS2DObject ) result.push( object );
+				if ( object.isCSS2DObject && object.autoSortZ ) result.push( object );
```

Comment:
Why not adding global property `renderer.sortObjects` and then do this?

```js
if ( renderer.sortObjects === true ) zOrder( scene );
```
That would be more in line with `WebGLRenderer` and `WebGPURenderer`.